.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
+notifications:
+  email: false
```

Comment:
I think email notifications would be helpful. It's not set to false for Smack and Openfire too.